bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/TapoControlHandlerFactory.java

```diff
@@ -42,15 +46,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
 /**
- * The {@link ThingHandler} is responsible for handling commands, which are
+ * The {@link TapoControlHandler} is responsible for handling commands, which are
```

Comment:
Wrong comment header, you should be presenting `TapoControlHandlerFactory` here.